swc/private/versions.bzl

```diff
@@ -5,6 +5,29 @@
 
 # Versions should be descending order so TOOL_VERSIONS[0] is the latest version.
 TOOL_VERSIONS = {
+    "v1.7.21": {},
```

Comment:
@jbedard yeah I see what you mean, sigh. I should turn off the automation until we can adjust for whatever change they made.

Reply:
Yeah. I ran the script myself as well and figured we should add everything up until it changed for now?

Reply:
I had hacked it to also add a lot of other versions (not just those on the first page of releases). Looks like you already found it: https://github.com/aspect-build/rules_swc/pull/272